src/common/api/accounts.ts

```diff
@@ -7,38 +7,38 @@ import type {
 import { fetchPendingTxs } from '@common/api/transactions';
 import { fetchFromSidecar } from '@common/api/fetch';
 
-export const fetchBalances = (apiServer: string) => (
-  principal: string
-): Promise<AddressBalanceResponse> => {
-  const path = `/address/${principal}/balances`;
-  return fetchFromSidecar(apiServer)(path);
-};
+export const fetchBalances =
+  (apiServer: string) =>
+  (principal: string): Promise<AddressBalanceResponse> => {
+    const path = `/address/${principal}/balances`;
+    return fetchFromSidecar(apiServer)(path);
+  };
```

Comment:
What's going on here? This isn't how prettier would format this?

Reply:
https://prettier.io/blog/2021/05/09/2.3.0.html

Reply:
cooo00ooOOool. Would be great to get this update added via the prettier-config though

Reply:
@kyranjamie what changes are required?